examples/vega-lite-examples.html

```diff
@@ -59,6 +60,9 @@ <h1>Vega-Lite Tooltip Examples</h1>
       addVlExample("specs/bar_layered_transparent.json", "#vis-layered-bar");
       addVlExample("specs/line_color.json", "#vis-color-line");
       addVlExample("specs/overlay_area_short.json", "#vis-overlay-area");
+      addVlExample("specs/bar_custom_render.json", "#vis-bar-custom-render", {
+        render: (value) => `The value of ${value.a} is ${value.b}`
```

Comment:
Should we use the sanitization argument here?

Reply:
I think the example implementation in any example should use sanitization so that we don't create unsafe HTML.